packages/projection-typeorm/src/operators/storeStakePoolMetadataJob.ts

```diff
@@ -4,6 +4,12 @@ import { STAKE_POOL_METADATA_QUEUE, StakePoolMetadataJob, defaultJobOptions } fr
 import { WithPgBoss } from './withTypeormTransaction';
 import { certificatePointerToId, typeormOperator } from './util';
 
+export const willStoreStakePoolMetadataJob = ({
+  eventType,
+  stakePools
+}: { eventType: ChainSyncEventType } & Mappers.WithStakePools) =>
+  eventType !== ChainSyncEventType.RollBackward && stakePools.updates.length > 0;
```

Comment:
For consistency I would change this in `eventType === ChainSyncEventType.RollForward`

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1200/commits/2af6d006fcc947890e054bbb1fc4f437f54c88cb